docs/elements/middleware.html

```diff
@@ -0,0 +1,131 @@
+        | App\Http\Middleware\VerifyCsrfToken       |
+        | App\Http\Middleware\ThrottleRequests      |"
+        php="
+    /**
+     * @Given there are middlewares disabled:
+     */
+    public function disableMiddlewares(TableNode $table): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>disableThrottling</code> method disables the throttling middleware. This is useful for testing
+        scenarios where rate limiting should be bypassed.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Given there's throttling middleware disabled"
+        php="
+    /**
+     * @Given there's throttling middleware disabled
+     */
+    public function disableThrottling(): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>disableCsrfProtection</code> method disables the CSRF protection middleware. This is useful for
+        testing scenarios where CSRF protection should be bypassed.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Given there's CSRF protection middleware disabled"
+        php="
+    /**
+     * @Given there's CSRF protection middleware disabled
+     */
+    public function disableCsrfProtection(): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>enableMiddleware</code> method enables a specified middleware. This is useful for scenarios where you
+        need to ensure that a particular middleware is active.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Given there is a 'App\Http\Middleware\Authenticate' middleware enabled"
+        php="
+    /**
+     * @Given there is a :middleware middleware enabled
+     */
+    public function enableMiddleware(string $middleware): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>requestShouldHaveUuidInField</code> method verifies that a specified field in the request contains a
+        valid UUID. This is useful for ensuring that UUIDs are correctly generated and included in the request.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then request object should have an UUID in 'transaction_id' field"
+        php="
+    /**
+     * @Then request object should have an UUID in :field field
+     */
+    public function requestShouldHaveUuidInField(string $field): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>middlewareShouldBeAppliedToRequest</code> method verifies that a specified middleware has been applied
+        to the request. This is useful for ensuring that the middleware is correctly modifying the request as expected.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the middleware 'App\Http\Middleware\Authenticate' should be applied to the request"
+        php="
+    /**
+     * @Then the middleware :middleware should be applied to the request
+     */
+    public function middlewareShouldBeAppliedToRequest(string $middleware): void"></code-snippet>
+    
```

Comment:
Here GitHub shows that there is no empty line at the end of the file.

![image](https://github.com/blumilksoftware/blt/assets/56546832/fe86dd7e-97f1-4719-ab9a-8410d72d6256)